docs/reference/migration/migrate_8_0/migrate_to_java_time.asciidoc

```diff
@@ -0,0 +1,315 @@
+[[migrate-to-java-time]]
+=== Java time migration guide
+
+With 7.0, {es} switched from joda time to java time for date-related parsing,
+formatting, and calculations. This guide is designed to help you determine
+if your cluster is impacted and, if so, prepare for the upgrade.
+
+
+
+[[java-time-convert-date-formats]]
+=== Convert date formats
```

Comment:
I'd add discrete tags. Otherwise, each of these headings creates its own page.
```suggestion
[discrete]
[[java-time-convert-date-formats]]
=== Convert date formats
```